cloudwatch/src/lambda_function.py

```diff
@@ -8,6 +8,9 @@
 
 KEY_INDEX = 0
 VALUE_INDEX = 1
+LOG_LEVELS = ['alert', 'trace', 'debug', 'notice', 'info', 'warn',
+          'warning', 'error', 'err', 'critical', 'crit', 'fatel',
```

Comment:
One of the log levels looks wrong, `fatel` should probably be `fatal`

```suggestion
          'warning', 'error', 'err', 'critical', 'crit', 'fatal',
```

Reply:
@vlj91 Thank you